src/models/Gladiator.js

```diff
@@ -0,0 +1,108 @@
+import { removeBuff } from "../actions";
+import { Dimachaerus, Secutor } from "../data/gladiatorData";
+
+
+import dimaWeapons from '../assets/images/weapons-Dimachaerus.png';
+import secutorWeapons from '../assets/images/weapons-Secutor.png';
+import retiWeapons from '../assets/images/weapons-Retiarius.png';
+
+import dimaAttack from '../assets/images/attack-Dimachaerus.png';
+import secutorAttack from '../assets/images/attack-Secutor.png';
+import retiAttack from '../assets/images/attack-Retiarius.png';
+
+import dimaAttackDW from '../assets/images/dwattack-Dimachaerus.png';
+
+export default class Gladiator {
```

Comment:
This isn't a perfect idea by any means, but it seems like there were a lot of values calculated on the fly for the render cycle portion of the Arena hook.

This sort of thing might help decouple that state and logic to come up with new values along the way and plug them in.

Also @patticus this image import at the top would be the way to bind these to a value and use them inline later.

As Gladiators got more specific, you could subclass them like this:

Secutor.js

```
export default class Secutor extends Gladiator {
  constructor(gladiator) {
    super(gladiator);
  }
  mySpecialAbility() {
  console.log('thats so special');
  }
}
```